website/docs/r/project_invitation.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: project_invitation"
+sidebar_current: "docs-mongodbatlas-resource-project-invitation"
+description: |-
+    Provides an Atlas Project Invitation resource.
+---
+
+# mongodbatlas_project_invitation
+
+`mongodbatlas_project_invitation` invites a user to join an Atlas project.
+
+Each Invitation for an Atlas User has a set of roles that provide access to a Project in an Organization.
+
+The roles that can be utilised can be found in the [MongoDB Documentation](https://docs.atlas.mongodb.com/reference/user-roles/#project-roles), which map to:
+
```

Comment:
resolved comment above doesn't seem to have been addressed.

Reply:
I am not sure what happened here, but fixing now.